File: codigos/codigo.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(void){
    int n;
    int tot = 0;

    cin >> n;

    vector<int> s(n);

    for (int i = 0; i < n; ++i) {
        cin >> s[i];
    }

    for (int i = 0; i <= n - 3; i++) {
        if (s[i] == 1 && s[i+1] == 0 && s[i+2] == 0) {
            tot++;
        }
    }

    cout << tot;
}

File: codigos/Python/codigo.py
def main():
    n = int(input())
    tot = 0

    s = [int(x) for x in input().split()]

    for i in range(n-2):
        if s[i] == 1 and s[i+1] == 0 and s[i+2] == 0:
            tot += 1

    print(tot)


if __name__ == "__main__":
    main()
